From 3cf0c775a10d011813f8bb7962e6e41a9757113a Mon Sep 17 00:00:00 2001
From: Spandan Das <spandandas@google.com>
Date: Tue, 22 Oct 2024 23:32:59 +0000
Subject: [PATCH 124/436] Mark android.hardware.drm@latest-service.clearkey as
 vendor

As part of the make to soong conversion, we are currently autogenerating
android_filesystem soong modules for each partition (system, system_ext,
vendor, ...), with the goal to eventually substitute the kati built
partitions. In order to generate the deps of these partitions, we are
currently using a heuristic to classify the modules in
`PRODUCT_PACKAGES` with the associated partition.

Mark the phony module `android.hardware.drm@latest-service.clearkey` as
`vendor: true` to help with this conversion heursitic.

Bug: 374371755
Test: verified that om.android.hardware.drm.clearkey.apex is present in the autogenerated soong-built
vendor partition

Change-Id: I4278e44b42b02877c46c66b3ddbe8e314369f270
---
 drm/mediadrm/plugins/clearkey/aidl/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drm/mediadrm/plugins/clearkey/aidl/Android.bp b/drm/mediadrm/plugins/clearkey/aidl/Android.bp
index cce6338a43..f46409f7c0 100644
--- a/drm/mediadrm/plugins/clearkey/aidl/Android.bp
+++ b/drm/mediadrm/plugins/clearkey/aidl/Android.bp
@@ -133,6 +133,7 @@ phony {
     required: [
         "com.android.hardware.drm.clearkey",
     ],
+    vendor: true,
 }
 
 cc_defaults {
-- 
2.43.0

